app/components/icons/logout-icon.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+function LogoutIcon() {
+  return (
+    <svg width="24" height="24" fill="none" viewBox="0 0 24 24">
```

Comment:
TIL that the icons are coming from https://iconic.app. Note to self, replace the other icons as well, as those were copied from figma.